### PR TITLE
[16.0][MIG] crm_lead_currency: Migration to 16.0

### PR DESCRIPTION
**Pull request by keylor2906**

### CODE REVIEW

**Comment by hugho-ad** [comment]
@luisg123v could you please review?

**Comment by hugho-ad** [comment]
@Josesosa07 

your input is welcome

File: crm_lead_currency/tests/test_crm_opportunity_currency.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import TransactionCase
```
Comment (luisg123v) [inline crm_lead_currency/tests/test_crm_opportunity_currency.py]:
```suggestion
from odoo.tests import TransactionCase
```

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: crm_lead_currency/tests/test_crm_opportunity_currency.py (hunk 1 above)
Comment (keylor2906) [reply crm_lead_currency/tests/test_crm_opportunity_currency.py]: Change applied.

**Review by luisg123v — APPROVED** [review]
LGTM :+1:

**Pull request merged**